pixmappy/PixelMapCollection.py

```diff
@@ -374,33 +411,49 @@ def __init__(self, name, pmap, projection, scale=1.):
         self.projection = projection
         self.scale = scale
         self.dest = None
-        return
+
     def reprojectTo(self, projection):
         self.dest = projection
-        return
-    def toSky(self, xy, c=None):
+
+    def toSky(self, x, y, c=None):
         ''' Return sky coordinates corresponding to array
         '''
-        return self.projection.toSky(self.pmap(xy,c) * self.scale)
-    def toPix(self, coords, c=None, guess=np.array([0.,0.])):
-        ''' Return pixel coordinates corresponding to input SkyCoords.
+        # This is the front end user interface.  Let's not modify the user's input x,y values.
```

Comment:
The toSky and toPix routines make a guarantee that the input x and y are not altered, but the PixelMap methods (__call__, inverse(), jacobian()) do not make this guarantee.  As I can see client calls making use of either set, would we want to be consistent about this?